components/Summary/Summary.jsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+
+import { Button } from 'components/Form';
+import SummaryList from 'components/Summary/SummaryList';
+import ErrorSummary from 'components/ErrorSummary/ErrorSummary';
+
+const MultiValue = (value) => (
+  <div key={value}>
+    <span>{value}</span>
+    <br />
+  </div>
+);
+
+export const SummarySection = ({
+  formData,
+  id,
+  title,
+  components,
+  formPath,
+}) => {
+  const Summary = (
+    <SummaryList
+      list={components
+        .filter(({ name }) => formData[name])
+        .map(({ name, label }) => ({
+          key: name,
+          title: label,
+          value: Array.isArray(formData[name])
+            ? formData[name]
+                .filter(Boolean)
+                .map((v) => MultiValue(v.split('/').pop()))
+            : typeof formData[name] === 'object'
+            ? Object.values(formData[name]).filter(Boolean).map(MultiValue)
+            : typeof formData[name] === 'boolean'
+            ? JSON.stringify(formData[name])
+            : formData[name],
+        }))}
+    />
+  );
+  return (
+    <div className="govuk-!-margin-bottom-7">
+      <h2>{title}</h2>
```

Comment:
Could you add the govuk-fieldset__legend--m className, so we get the correct font

Reply:
Good idea, but it's not a `fieldset`, let me see if I can find something similar 🤔

Reply:
I am going to use `govuk-heading-m and making the element `h3` so it's more semantic 👍